lib/features/label-editing/ChoreoLabelEditingProvider.js

```diff
@@ -0,0 +1,74 @@
+import {
+  assign,
+  pick
+} from 'min-dash';
+
+import {
+  is
+} from 'bpmn-js/lib/util/ModelUtil';
+import inherits from 'inherits';
+import LabelEditingProvider from 'bpmn-js/lib/features/label-editing/LabelEditingProvider';
+import { heightOfBottomBands, heightOfTopBands } from '../../util/BandUtil';
+
+export default function ChoreoLabelEditingProvider(eventBus, canvas, directEditing,
+    modeling, resizeHandles, textRenderer) {
+  LabelEditingProvider.call(this, eventBus, canvas, directEditing,
+    modeling, resizeHandles, textRenderer);
+  //directEditing.registerProvider(this);
+  this._modeling = modeling;
+  this._canvas = canvas;
+}
+
+inherits(ChoreoLabelEditingProvider, LabelEditingProvider);
+
+
+ChoreoLabelEditingProvider.$inject = [
```

Comment:
Fun fact: Christian found out that providing this array is actually unnecessary, because the injector parses the names of the constructor parameters. We can maybe add that to the refactoring-when-we-got-time-at-some-point list.